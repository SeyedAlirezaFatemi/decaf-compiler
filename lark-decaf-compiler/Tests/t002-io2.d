int main() {
    int a;
    int b;
    int i;

    b = 0;
    for(i = 1; true; i = i + 1) {
        Print("Please enter the #", i, " number:");
        a = ReadInteger();
        if (a < 0)
            break;
        b = b + a;
    }

    Print("Sum of ", i, " items is: ", b);
}