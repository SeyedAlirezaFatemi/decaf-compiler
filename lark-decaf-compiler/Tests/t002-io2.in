6
95
49
-85
-54
95
48
48
5484
48
-0705
-2540
56418
6484
840
-5
