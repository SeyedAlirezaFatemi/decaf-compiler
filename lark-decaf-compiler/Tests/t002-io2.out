Please enter the #1 number:
Please enter the #2 number:
Please enter the #3 number:
Please enter the #4 number:
Sum of 4 items is: 150
